website/js/components/insilicopred/SplicingLevelSubtile.js

```diff
@@ -0,0 +1,455 @@
+                </tbody>
+            </table>
+        );
+    }
+}
+
+class SpliceSiteImpactTable extends React.Component {
+    render() {
+        const {prior, type, variantZScore, data} = this.props;
+
+        let extraMessage = null;
+
+        if (prior === 0.34) {
+            if (type === 'donor' && variantZScore < -2.0) {
+                /* special case #2: native donor (when type == 'donor') */
+                extraMessage = "This variant was classified as Moderate because the Z-score for the wild type is already low (less than -2) and the change due to variation is not considered large enough to warrant Severe.";
+            }
+            else if (type === 'acceptor' && variantZScore < -1.5) {
+                /* special case #3: native acceptor (when type == 'acceptor') */
+                extraMessage = "This variant was classified as Moderate because the Z-score for the wild type is already low (less than -1.5) and the change due to variation is not considered large enough to warrant Severe.";
+            }
+        }
+
+        return (<div>
+            <div style={{textAlign: 'center', fontWeight: 'bold', margin: '20px'}}>
+                Probability of Pathogenicity<br />
+                [Due to Splice {capitalize(type)} Damage]
+            </div>
+
+            <table className="splicing-level-table">
+                <tr>
+                    <td width="60%">Qualitative Category</td>
+                    <td width="30%">Z-score Range</td>
+                    <td width="20%">Probability</td>
+                </tr>
+
+                {
+                    // FIXME: highlight row that matches the prior prob value
+                    splicingImpactFields.fields.map(x =>
+                        <tr key={x.key} className={prior === x.prob && (!x.check || x.check(data)) ? 'highlighted' : ''}>
+                            <td className={`pathos-prob-label-${x.key}`}>{x.label}</td>
+                            <td>{splicingImpactFields.zScoreLabels[type][x.key]}</td>
+                            <td>{x.prob}</td>
+                        </tr>
+                    )
+                }
+            </table>
+
+            { extraMessage && <div>{ extraMessage }</div> }
+        </div>);
+    }
+}
+
+class DeNovoDonorPathogenicityTable extends React.Component {
+    render() {
+        const {prior, data} = this.props;
+
+        return (<div>
+            <div style={{textAlign: 'center', fontWeight: 'bold', margin: '20px'}}>
+                Probability of Pathogenicity<br />
+                [Due to De Novo Donor]
+            </div>
+
+            <table className="splicing-level-table">
+                <tr>
+                    <td width="60%">Qualitative Category</td>
+                    <td width="30%">Z-score Range</td>
+                    <td width="20%">Probability</td>
+                </tr>
+
+                {
+                    // FIXME: highlight row that matches the prior prob value
```

Comment:
same

Reply:
Same as the above. In this case, though, we're unfortunately still missing the condition for prob == 0.02. I've added a FIXME to that collection indicating where the check conditions should go.